Task5/BookCollection/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using BookCollection.Contracts;
+
+namespace BookCollection
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            ICatalog catalog = new Catalog();
+
+            Book book1 = BookFactory.CreateBook("Book One", new DateTime(2001, 5, 20), new List<string> { "Author A", "Author B" });
+            Book book2 = BookFactory.CreateBook("Book Two", new DateTime(1999, 8, 15), new List<string> { "Author A" });
+            Book book3 = BookFactory.CreateBook("Book Three", new DateTime(2010, 3, 10), new List<string> { "Author C" });
+
+            catalog.AddBook("123-4-56-789012-3", book1);
+            catalog.AddBook("1234567890123", book2);
+            catalog.AddBook("456-7-89-012345-6", book3);
```

Comment:
ISortStrategy is not needed, because it is implemented in Catalog. Good practice but not needed
SortByPublicationDateStrategy class not used at all